rest/src/test/java/config/RestTestConfig.java

```diff
@@ -0,0 +1,63 @@
+package config;
+
+import com.carhouse.service.*;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+import javax.sql.DataSource;
+
+import static org.mockito.Mockito.mock;
+
+@Configuration
+@ComponentScan({"com.carhouse.rest.controller", "com.carhouse.rest.handler", "com.carhouse.rest.response"})
+public class RestTestConfig {
```

Comment:
Why did you decide to have all mocks in one place and load all these beans in context when they are not in use?
It would be better to mock separately

Reply:
I agree with you, I've changed it